sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -1324,6 +1360,7 @@ pi_result piEnqueueMemBufferReadRect(pi_queue, pi_mem, pi_bool,
 pi_result piEnqueueMemBufferWrite(pi_queue, pi_mem, pi_bool, size_t, size_t,
                                   const void *, pi_uint32, const pi_event *,
                                   pi_event *) {
+  // TODO : intel/llvm_test_suite
```

Comment:
what does this mean - "// TODO add a test to github.com/intel/llvm-test-suite"?
Please update if so. Even though I don't see much sense in such TODO.

Reply:
Will do.